DVR/SessionDataTask.swift

```diff
@@ -30,67 +30,70 @@ class SessionDataTask: NSURLSessionDataTask {
     }
 
     override func resume() {
-        let cassette = session.cassette
+        let queue = dispatch_queue_create("com.venmokit.DVR.sessionDataTaskQueue", nil)
+        dispatch_async(queue) { [unowned self] in
```

Comment:
I don't know if either is better, TBH. I think doing all this work on another queue might better simulate a network request.

Just wanted to raise my thought process, honestly.